src/jit/bitsetasshortlong.h

```diff
@@ -513,11 +513,10 @@ class BitSetOps</*BitSetType*/ BitSetShortLongRep,
 #endif
 
                 // If there's a bit, doesn't matter if we're short or long.
-                if (hasBit)
+                if (m_bits != 0)
                 {
```

Comment:
It would be useful to add a comment here about why you're checking `m_bits != 0` instead of `hasBits`.